src/cdefs-compat.h

```diff
@@ -14,21 +14,27 @@
 #ifdef __GNUC__
 #ifndef __strong_reference
 #ifdef __APPLE__
-#define __strong_reference(sym,aliassym) __weak_reference(sym,aliassym)
+#define __strong_reference(sym,aliassym) __weak_reference_compat(sym,aliassym)
 #else
 #define __strong_reference(sym,aliassym)	\
 	OLM_DLLEXPORT extern __typeof (sym) aliassym __attribute__ ((__alias__ (#sym)));
 #endif /* __APPLE__ */
 #endif /* __strong_reference */
 
 #ifdef __wasm__
-#  define __weak_reference(sym,alias) __strong_reference(sym,alias)
+#  define __weak_reference_compat(sym,alias) __strong_reference(sym,alias)
 #endif
 
-#ifndef __weak_reference
+#ifdef __weak_reference
+#ifdef __NetBSD__
+#define __weak_reference_compat(sym,alias) __weak_reference(sym)
```

Comment:
```suggestion
#define __weak_reference_compat(sym,alias) __weak_alias(alias,sym)
```

Reply:
Thank you. Is it okay to check for __weak_reference regardless or should we change this to something like

```
#if defined(__weak_alias) && defined(__NetBSD__)
...
#elif defined(__weak_reference)
...
```